prow/plugins/pony/pony.go

```diff
@@ -125,34 +125,37 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, p
 		return nil
 	}
 	// Make sure they are requesting a pony
-	mat := match.FindStringSubmatch(e.Body)
-	if mat == nil {
+	mats := match.FindAllStringSubmatch(e.Body)
+	if mats == nil {
 		return nil
 	}
 
-	tag := mat[1]
-	org := e.Repo.Owner.Login
-	repo := e.Repo.Name
-	number := e.Number
-
-	for i := 0; i < 5; i++ {
-		resp, err := p.readPony(tag)
-		if err != nil {
-			log.WithError(err).Println("Failed to get a pony")
-			continue
+	for _, mat := range mats {
```

Comment:
Can we have a limit for the maximum number of pony requests per comment?

Reply:
Thank you for your comments @alvaroaleman, What is the right number for max ponies in a comment, I was thinking 5. Should I go ahead with 5?

Reply:
Five sounds good to me